sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/jaas/JaasResolver.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.implementation.jaas;
+
+import static com.azure.spring.cloud.service.implementation.kafka.AzureKafkaPropertiesUtils.convertJaasStringToMap;
+
+public final class JaasResolver {
+
+    private JaasResolver() {
+    }
+
+    public static Jaas resolve(String jaasConfig) {
+        String[] arr = extractLoginModuleAndControlFlagFromJaasString(jaasConfig);
+        Jaas jaas = new Jaas(arr[0], arr[1]);
+        jaas.setOptions(convertJaasStringToMap(jaasConfig));
```

Comment:
Why is the `convertJaasstringToMap` not in this resolver?

Reply:
It has been moved into resolver in the last commit.